src/service-metrics.js

```diff
@@ -49,15 +49,18 @@ var serviceMatchers = {
 	'markets': /^http:\/\/markets\.ft\.com/,
 	'fastly': /^https?:\/\/next\.ft\.com/,
 	'fastly-api': /^https:\/\/api\.fastly\.com/,
-	'harrier': /^https:\/\/ft-next-harrier-eu\.herokuapp\.com\//,
-	'personalised-feed-v1': /^http:\/\/(personalised-feed\.ft\.com|ft-next-personalised-feed-api\.herokuapp\.com)\/v1\/feed/,
+	'ft-next-harrier-eu': /^https:\/\/ft-next-harrier-eu\.herokuapp\.com\//,
+	'ft-next-personalised-feed-api': /^http:\/\/(personalised-feed\.ft\.com|ft-next-personalised-feed-api\.herokuapp\.com)\/v1\/feed/,
 	'graphite': /^https:\/\/www\.hostedgraphite\.com\//
 };
 
 module.exports = {
 	init: function (additionalServices) {
 		if (additionalServices) {
-			Object.keys(additionalServices).forEach(function (serv) {
+			Object.keys(additionalServices).forEach(function(serv) {
+				if (serv.toLowerCase() !== serv || serv.indexOf('.') > -1) {
```

Comment:
I would also add a warning in here to say this functionality is deprecated and they should just add the service to next-express, then when we move all this to service-registry most of it will already be in one place